campaigns/windows-10/logo-swap.html

```diff
@@ -0,0 +1,230 @@
+           });
+
+           var check_default_timeout = window.setInterval(function () {
+               if (poll_retry > 500) {
+                   clearInterval(check_default_timeout);
+               }
+               poll_retry++;
+               isDefault(function yesDefault () {
+                   //Default has switched while the snippet is opened
+                   if (is_firefox_default !== true) {
+                       showSuccessContent();
+                       is_firefox_default = true;
+                       if (cta_clicked === true) {
+                           sendMetric('conversion-is-default-changed-after-click');
+                       } else {
+                           sendMetric('conversion-is-default-changed-without-click');
+                       }
+                       clearInterval(check_default_timeout);
+                   }
+               }, function noDefault () {
+                   if (is_firefox_default !== false) {
+                       showNonDefaultContent();
+                       is_firefox_default = false;
+                   }
+               });
+           }, 1000);
+
+           function showDefaultContent () {
+               snippet_text_default.style.display = 'block';
+               snippet_text_success.style.display = 'none';
+               snippet_content_default.style.display = 'block';
+               snippet_content_not_default.style.display = 'none';
+               snippet_icon.src = '{{ icon }}';
+
+               for (var i=0, k=buttons.length; i<k; i++) {
+                   buttons[i].addEventListener('click', function (e) {
+                       e.preventDefault();
+                       sendMetric('conversion-is-default-click', function () {
+                           window.location = '{{ url_default|safe }}';
+                       });
+                   }, false);
+               }
+           }
+
+           function showSuccessContent () {
+               snippet_text_default.style.display = 'none';
+               snippet_text_success.style.display = 'block';
+               snippet_content_default.style.display = 'block';
+               snippet_content_not_default.style.display = 'none';
+               snippet_icon.src = '{{ icon }}';
+
+               for (var i=0; i < buttons.length; i++) {
+                   buttons[i].addEventListener('click', function (e) {
+                       e.preventDefault();
+                       sendMetric('conversion-is-default-click', function () {
+                           window.location = '{{ url_default|safe }}';
+                       });
+                   }, false);
+               }
+           }
+
+           function showNonDefaultContent () {
+               snippet_content_default.style.display = 'none';
+               snippet_content_not_default.style.display = 'block';
+
+               for (var i=0; i < buttons.length; i++) {
+                   buttons[i].addEventListener('click', function (e) {
```

Comment:
For example here you want to capture `click` to do other things apart from sending the metric. Change the code to
```javascript
               for (var i=0; i < buttons.length; i++) {
                   buttons[i].dataset.metric = 'conversion-is-default-click';
               }

               for (var i=0; i < buttons.length; i++) {
                   buttons[i].addEventListener('click', function (e) {
                       e.preventDefault();
                       Mozilla.UITour.setConfiguration('defaultBrowser');
                       cta_clicked = true;
                   }, false);
               }
```